HdProduction.HelpDesk.Domain/Entities/Comment.cs

```diff
@@ -1,19 +1,26 @@
+using System.Collections.Generic;
+
 namespace HdProduction.HelpDesk.Domain.Entities
 {
   public class Comment : EntityBase<long>
   {
-    public Comment(long ticketId, string text, long userId)
+    public Comment(long ticketId, string text, long userId, long? replyToComment)
     {
       TicketId = ticketId;
       Text = text;
       UserId = userId;
+      ReplyToComment = replyToComment;
     }
 
     public long TicketId { get; }
     public string Text { get; }
     public long UserId { get; }
+    public long? ReplyToComment { get; }
```

Comment:
ReplyToComment**Id**? Cause it looks like entity.

Reply:
fixed